src/content/docs/en/guides/deploy/zeabur.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Deploy your Astro Site to Zeabur
+description: How to deploy your Astro site to the web on Zeabur.
+type: deploy
+i18nReady: true
+---
+[Zeabur](https://zeabur.com) offers hosting for full-stack web applications. Any Astro site can be hosted on Zeabur as an SSR or static site.
+
+This guide includes instructions for deploying to Zeabur through the website UI.
+
+## Project Configuration
+
+Your Astro project can be deployed to Zeabur in two different ways: as a static site, or a server-rendered site.
```

Comment:
I think we can drop this sentence, since it's basically a repetition of the first sentence in the guide.